### ISSUE TITLE
Where is the project directory? I couldn't find the .atom-live-server.json

### ISSUE THREAD

**Issue by kolastyl** [jas-chen/atom-live-server#165]
Where exactly is the project directory? Is it within the .atom folder?

I want to use the atom-live-server package on a browser different from my default, say Firefox Developer Edition. I couldn't configure it to work using guidelines seen online, please help. Am I to manually create the .json file? If yes, should it be at the root folder of my preferred directory? Please safe a life.

Kolastyl

**Comment by jas-chen** [comment]
yes, you have to manually create the .json file and save it at the root of the project.

**Comment by jwhitleybb** [comment]
I have created the .atom-live-server.json file in the project root folder but the server still goes to the ~/ directory. Is this wrong?

`{
  "root": "/Users/username/Desktop/project-folder/"
}`

**Comment by alexxtasi** [comment]
Hi all
trying to do the same thing, add ```.atom-live-server.json``` in my project's directory 
```
{
  "root": "/home/user/project/"
}
```
it keeps running live server from the directory which is first in the project's list.

Am I missing something here ?
regards

EDIT:
oups... maybe I am missing smth here...
Based on [live server](https://github.com/tapio/live-server#usage-from-node)'s param list, ```root``` is not the project dir. Is that right ?
